### PR TITLE
QA Web playbook

### PR DESCRIPTION
**Pull request by sbesson**
This PR defines a new playbook for deploying nodes used for the Web applications (registry, QA). Depending on their usage (staging/production), they can be used with/without `ci-provision.yml` to hook them to Jenkins. /cc @kennethgillen

### CODE REVIEW

File: ansible/README.md [hunk 1]
```diff
@@ -58,9 +58,10 @@ As another alternative you can probably install ansible into your virtualenv (un
 Examples
 --------
 
-In the following examples replace example-hosts with the private host inventory file
+In the following examples replace example-hosts with the private host
+inventory file
```
Comment (manics) [inline ansible/README.md]: Line wars! I've been working on the basis that inserting line-breaks is a pain.

Comment (sbesson) [inline ansible/README.md]: If that's the only issue with this PR, I am happy to revert it

Comment (sbesson) [inline ansible/README.md]: https://github.com/sbesson/infrastructure/commit/e3b79ab03d8cf715227aa541da2b612d7f0b4e45

Comment (manics) [inline ansible/README.md]: Cheers. Didn't want to get into a situation where everyone's doing different things. Rest of the PR looks OK

**Comment by manics** [comment]
Ooops...the nginx role wasn't tested properly when it was merged :) It relies on `yum install epel-release` having been previously run, so it should work fine on our LSC VMs which might have epel enabled, but not on the docker container I used for testing. Apart from that it works fine.

What do you think- install `epel-release`? Or deal with separately along with the other clean-ups/improvements?

**Comment by sbesson** [comment]
I think the main target of this new playbook are LSC VMS right @kennethgillen? If so, I would defer the installation/enabling of `epel-release` to a cleanup PR.

**Comment by manics** [comment]
ok, want me to merge then?

**Comment by sbesson** [comment]
I assume this will be tested on the new QA Web VMs that will be spinned up so I defer to Kenny no whether we should merge now or later.

File: ansible/qa-web-deployment.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+# Playbook for provisioning QA Web nodes
+
+- hosts: qa-web-linux
+  roles:
+    - role: postgresql
+    - role: nginx
+  vars:
+    - javajdkversions:
```
Comment (manics) [inline ansible/qa-web-deployment.yml]: You don't need these vars unless you're installing java

**Comment by sbesson** [comment]
Merging. Provisioning issues to be opened.

**Pull request merged**